sky/packages/sky/lib/src/fn3/app.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:sky' as sky;
+
+import 'package:sky/material.dart';
+import 'package:sky/painting.dart';
+import 'package:sky/src/fn3/basic.dart';
+import 'package:sky/src/fn3/binding.dart';
+import 'package:sky/src/fn3/framework.dart';
+import 'package:sky/src/fn3/navigator.dart';
+import 'package:sky/src/fn3/theme.dart';
+import 'package:sky/src/fn3/title.dart';
+
+const TextStyle _errorTextStyle = const TextStyle(
+  color: const Color(0xD0FF0000),
+  fontFamily: 'monospace',
+  fontSize: 48.0,
+  fontWeight: FontWeight.w900,
+  textAlign: TextAlign.right,
+  decoration: underline,
+  decorationColor: const Color(0xFFFF00),
+  decorationStyle: TextDecorationStyle.double
+);
+
+class App extends StatefulComponent {
+  App({
+    Key key,
+    this.title,
+    this.theme,
+    this.routes
+  }): super(key: key);
+
+  final String title;
+  final ThemeData theme;
+  final Map<String, RouteBuilder> routes;
+
+  AppState createState() => new AppState();
+}
+
+class AppState extends State<App> {
+
+  GlobalObjectKey _navigator;
+
+  void initState(BuildContext context) {
```

Comment:
We should remove the `context` parameter to this function now that there's a `context` getter on `State`.